<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	border : 4px solid rgb(45, 45, 45);
	border-radius:20px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget_2{
	background-color: rgb(195, 195, 195);
	border-top-left-radius:20px;
	border-top-right-radius:20px;
}
QPushButton{
	background-color :rgba(0, 0, 0, 0);
	color: rgb(144, 144, 144);
	font:bold;
	font size:15px;
	font family : entypo;
}

QPushButton:hover{
	color:rgb(	142, 176, 27);
}

QPushButton:pressed{
	color:rgb(	91, 88, 53);
	padding-top : 5px;
	padding-left : 5px;
}


</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(142, 176, 27);
border-radius:7px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(45, 45, 45);
border-radius:7px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(142, 176, 27);
border:4px solid rgb(45, 45, 45);
border-radius:7px;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16000</width>
               <height>16000</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">colour:rgb(144, 144, 144);

background-color: rgb(0, 170, 0);
background-color: rgb(0, 85, 0);
background-color: rgb(0, 0, 127);
background-color: rgb(89, 102, 52);</string>
             </property>
             <property name="text">
              <string>VirginieBROWSER</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/.iconify/font-awesome/regular/window-minimize.svg</normaloff>:/.iconify/font-awesome/regular/window-minimize.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/.iconify/font-awesome/regular/window-maximize.svg</normaloff>:/.iconify/font-awesome/regular/window-maximize.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">Button:hover{
	background-color: rgb(255, 50, 35);
	color: rgb(255, 69, 52);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/.iconify/font-awesome/regular/window-close.svg</normaloff>:/.iconify/font-awesome/regular/window-close.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/.iconify/font-awesome/regular/arrow-alt-circle-left.svg</normaloff>:/.iconify/font-awesome/regular/arrow-alt-circle-left.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/.iconify/font-awesome/regular/arrow-alt-circle-right.svg</normaloff>:/.iconify/font-awesome/regular/arrow-alt-circle-right.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/Downloads/refresh (2).png</normaloff>:/Downloads/refresh (2).png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(31,31,31);
border-radius:5px;
color : rgb(144, 144, 144);
padding-left:5px;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(	45, 45, 45);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(45, 45, 45);
border-bottom-left-radius:20px;
border-bottom-right-radius:20px;
color:rgb(144, 144,144);</string>
        </property>
        <property name="text">
         <string>Developed by Virginie</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myressources.qrc"/>
 </resources>
 <connections/>
</ui>
